src/Aspose.Imaging.Cloud.Sdk.Test/Base/ApiTester.cs

```diff
@@ -117,13 +119,11 @@ public abstract class ApiTester
         protected readonly string[] BasicExportFormats = new string[]
         {
             "bmp",
-            "jpg",
-            "psd",
-            "tiff",
             "gif",
+            "jpg",
             "png",
-            "j2k",
```

Comment:
@MaxCrank Are you sure that need to remove j2k, webp formats from tests?

Reply:
@roxboris please check corporate mail regarding tests across all levels.